code/__DEFINES/conflict.dm

```diff
@@ -77,6 +77,7 @@
 #define GUN_AMMO_COUNT_BY_PERCENTAGE (1<<11)
 #define GUN_AMMO_COUNT_BY_SHOTS_REMAINING (1<<12)
 #define GUN_NO_PITCH_SHIFT_NEAR_EMPTY (1<<13)
+#define ITEM_SHOWS_AMMO_REMAINING (1<<14)
 
```

Comment:
not an item flag but named item

Reply:
What's a better way to name it? I said item because it's the item_state that it effects.

Reply:
updated